prijateli_tree/app/main.py

```diff
@@ -42,17 +41,44 @@ def route_admin_access():
 
 
 @app.post("/game/")
-def route_create_game(game_type: int, user_id: int, num_rounds: int, practice: bool):
-    new_game_id = create_new_game(game_type, user_id, num_rounds, practice)
-    return {"status": "success", "game_id": new_game_id}
+def route_create_game(
+    game_data: GameCreate,
+    db: Session = Depends(get_db),
+):
+    new_game = Game(
+        created_by=game_data.created_by,
```

Comment:
Right now we don't have this data available, it'll come from the logged in user once that's in place. Just use the value `1` for the time being.

Reply:
Where shoud I use '1'?

Reply:
It's just a stop-gap since `game_data.created_by` won't be an actual value, at at least for a new game. The `1` value represents a generic user ID.